recipe/provision.php

```diff
@@ -63,19 +67,20 @@
         'public_path',
         'php_version',
         'db_type',
+    ];
+    $dbparams = [
         'db_user',
         'db_name',
         'db_password',
     ];
-    $code = "\n\n    host(<info>'{{alias}}'</info>)";
-    foreach ($params as $name) {
```

Comment:
Should we keep printing those code parts?

Reply:
Looks like it was left over from development.

Always prints this:
``` bash

host('your_hostname_here')
 ->set('sudo_password', '…')
 ->set('domain', '…')
 ->set('public_path', '…')
 ->set('php_version', '…')
 ->set('db_type', '…')
 ->set('db_user', '…')
 ->set('db_name', '…')
 ->set('db_password', '…');

```

Reply:
> Should we keep printing those code parts?

Hi @antonmedv, I just noticed this code was put back in the master. Just curious, what's the purpose of printing those in the output?

Thanks!

Reply:
So users can cope pase those and put into deploy.php.

Reply:
If that's the case, I want to point out couple things.

We are printing lines for only $params array now. I split the `$params` array to `$dbparams` to aid in asking database info only if the database type is not `none`

So, if you need all the parameters output like before, I would modify it to:


```php


    $params = [
        'sudo_password',
        'domain',
        'public_path',
        'php_version',
        'db_type',
    ];
    $dbparams = [
        'db_user',
        'db_name',
        'db_password',
    ];
    $code = "\n\n    host(<info>'{{alias}}'</info>)";
    foreach (array_merge($params, $dbparams) as $name) {
        $code .= "\n        ->set(<info>'$name'</info>, <info>'…'</info>)";
    }
    $code .= ";\n\n";
    writeln($code);
});


```

In this, the `array_merge` function will combine `$params` and `$dbparams` into a single array, which is then iterated over to generate the configuration code. This way, all parameters from both arrays will be included in the output.

---

Also, I think it would be helpful to include a message just before printing it. 

For example, "To avoid entering parameters during script execution, include the following config with values in your deploy.php"

---

Let me know what you think.

Reply:
Good suggestions. PR?